modules/tb-all-in-one/user_data.sh

```diff
@@ -1,21 +1,40 @@
-#!/bin/bash
+#!/bin/bash -ex
 set -euo pipefail
 
+exec > >(tee /var/log/user-data.log|logger -t user-data -s 2>/dev/console) 2>&1
```

Comment:
oh, it means, take everything from the rest of this script and dump it into this pipe #bash

Reply:
TIL